cmd/kubepkg/templates/latest/rpm/kubelet/kubelet.spec

```diff
@@ -43,13 +46,18 @@ install -p -m 755 -t %{buildroot}%{_bindir}/ kubelet
 install -p -m 644 -t %{buildroot}%{_unitdir}/ kubelet.service
 install -m 755 -d %{buildroot}%{_sysconfdir}/sysconfig/
 install -p -m 644 -T kubelet.env %{buildroot}%{_sysconfdir}/sysconfig/kubelet
+install -m 755 -d %{buildroot}%{_sysconfdir}/cni/net.d/
+install -m 755 -d %{buildroot}/opt/cni/bin
+mv cni-plugins/* %{buildroot}/opt/cni/bin/
 
 # TODO: Do we need this?
 #%make_install
 
 %files
 %{_bindir}/kubelet
 %{_unitdir}/kubelet.service
+%{_sysconfdir}/kubernetes/manifests/
+/opt/cni
```

Comment:
As mentioned in https://github.com/kubernetes/release/pull/1309#discussion_r432943531, I think we could discuss a better location than `/opt`